source/content/nutzen/api-nutzen.rst

```diff
@@ -1,154 +1,473 @@
-funktioniert, erklären wir Ihnen in der folgenden Auflistung, mit welcher Terminologie Sie welche
-Einheiten in opendata.swiss erreichen können:
+Über die API können Sie nach sämtlichen Metadaten in opendata.swiss suchen. Die Suchsyntax entspricht Apache `Lucene <https://lucene.apache.org/>`__, da opendata.swiss auf CKAN basiert und Apache `Solr <https://solr.apache.org/>`__ als Suchmaschine benutzt. Dies ist ebenfalls in der  `CKAN-Dokumentation <https://docs.ckan.org/en/latest/user-guide.html#search-in-detail>`__ ausführlich beschrieben. 
+Arbeiten Sie mit Metadaten der Organisation, Metadaten der Datasets oder Metadaten der Ressourcen? Beachten Sie dabei die Ebenen: Die folgende Auflistung zeigt die wichtigsten Terminologien, mit welchen Sie Informationen in opendata.swiss finden können:
+
 
+``organization``
+   Entspricht der `datenpublizierenden Organisation <https://opendata.swiss/de/organization>`__. Jedes Dataset wird von genau einer Organisation publiziert. 
+   Eine Organisation kann jedoch Unterorganisationen haben.
 ``package``, ``dataset``
    Entspricht einem :term:`Dataset <Dataset>` bei opendata.swiss. Damit gemeint sind die Metadaten
    zu einem Bündel von Ressourcen.
-``resource``
-   Entspricht einer Ressource bei Opendata.swiss. Die Plattform opendata.swiss hostet
-   selbst keinen Ressourcen, stellt aber «downloadUrls» oder «accesssUrls»
-   und teilweise auch Daten Previews zur Verfügung.
-   Ressourcen in CKAN entsprechen dcat:Distribution im :doc:`DCAT AP Standard <../glossar/bibliothek/dcat-ap-ch>`
+``id``
+   Entspricht in der Regel dem :term:`Slug <Slug>` des Datasets oder der Organisation auf opendata.swiss. 
+   Bei Ausnahmefällen konsultieren Sie bitte die CKAN-Dokumentation.
+``resource``, ``distribution``
+   Entspricht einer `Ressource <https://handbook.opendata.swiss/de/content/glossar/begriffe.html#term-Distribution>`__ bei Opendata.swiss. Die Plattform opendata.swiss hostet selbst keine Datasets oder Ressourcen, stellt aber downloadUrls oder accesssUrls und teilweise auch Daten Previews zur Verfügung. Ressourcen in CKAN entsprechen dcat:Distribution im :doc:`DCAT AP Standard <../glossar/bibliothek/dcat-ap-ch>`.
 ``group``
    Entspricht einer Kategorie bei opendata.swiss. Ein Dataset kann mehreren Kategorien angehören.
-``organization``
-   Entspricht den Datenpublizierenden, egal ob Person oder Organisation.
-   Jedes Dataset wird von genau einer Organisation publiziert.
-``id``
-   Entspricht im API dem :term:`Slug <Slug>` des Datasets.
+   Entspricht dcat:theme im :doc:`DCAT AP Standard <../glossar/bibliothek/dcat-ap-ch>`
+
 
 .. _api-nutzen-beispiele:
 
-Beispiele
-^^^^^^^^^^
+Anwendungsbeispiele für die API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typische Abfragen für Datennutzende
+-----------------------------------
+
+.. toggle-header::
+    :header: status_show
+
+    .. code-block:: bash
+      :caption: Status der Plattfrom
```

Comment:
Would it make more sense to write as a header what exactly API request should do:
  :header: status_show -> :header: Status der Plattform
  and then one can see how to make this kind of request